libs/coin-modules/coin-algorand/package.json

```diff
@@ -72,7 +72,7 @@
   "scripts": {
     "clean": "rimraf lib lib-es",
     "build": "tsc && tsc -m ES6 --outDir lib-es",
-    "coverage": "jest --coverage=true && mv coverage/coverage-final.json coverage/coverage-algorand.json",
+    "coverage": "jest --coverage --collectCoverageFrom='src/**/*.{ts,tsx}' --testPathIgnorePatterns='/bridge.integration.test.ts|node_modules|lib-es|lib/' --passWithNoTests && mv coverage/coverage-final.json coverage/coverage-algorand.json",
```

Comment:
`--collectCoverageFrom='src/**/*.{ts,tsx}'` this line should be unnecessary as `jest.config.js` file already define it (and there is no *tsx* file).

Reply:
Alright, removing it 👌